File: variables-y-tipo-de-datos.js
// Tipo de Datos y Variables
/* 

String
Number
Boolean
Undefined

Object:
    Null
    Literal Object
    Array

function

*/

let nombre = "";
let apellido = '';
let direccion = `Lorem ipsum dolor sit amet consectetur adipisicing
 elit. Possimus minima fuga nam ${nombre} inventore ${apellido} dolor adipisci reiciendis
  praesentium dolorum obcaecati, veniam pariatur. Est, quisquam? 
  Libero autem animi temporibus porro enim quia!`; // string

let edad = 40;
let precio = 10.50;
let grados = -10;
let saldo = -1.5; // number

let active = true; // boolean 
let open = false; // boolean

let peso; // undefined
let users = null; // object 


let persona = {
    nombre: "",
    apellido: ""
}

persona.nombre = "John"
persona["apellido"] = "Doe"

let nombres = ["Hugo", "Paco", "Luis"];
nombres[1] = "Donald"

nombres.push("Mickey")

function nombreFuncion(){

}

const nombreFuncion2 = function(){
    
}

File: variables-y-tipo-de-datos.py
# Tipo de Datos y Variables
""" 
String
Int
Float
Boolean

None

Dict

Array:
    List
    Tuple
    Set

func

"""

nombre = ""
apellido = ''

direccion = f"""Lorem ipsum dolor sit amet consectetur adipisicing
 elit. Possimus minima fuga nam {nombre} inventore {apellido} dolor adipisci reiciendis
  praesentium dolorum obcaecati, veniam pariatur. Est, quisquam? 
  Libero autem animi temporibus porro enim quia!""" # str
  
edad = 40 # int
precio = 10.50 # float
grados = -10 # int
saldo = -1.5 # float

active = False
open = True

peso = None
users = None

persona = {
    "nombre": "",
    "apellido": ""
}

persona["nombre"] = "Jane"
persona["apellido"] = "Doe"

# List
nombres = ["Hugo", "Paco", "Luis"] # list
nombres[1] = "Donald"
nombres.append("Mickey")

# Tuple
status = ("active", "inactive", "suspended", "canceled", "approved", "banned")
status[3]

# Set
frutas = { "Pera", "Manzana", "Uva", "Banana"}


def nombre_funcion():
    pass

sumar = lambda a, b: a + b